### PR TITLE
Add russian language

### PR DESCRIPTION
**Pull request by NaoCraftLab**

### CODE REVIEW

**Comment by DarthSidiousPalpatine** [comment]
Why you make unnecessary capital letters and why you translate multiblock as мультиблок. I mean, "Чтобы увидеть этот мультиблок" sounds SOOOO weird. Use Структура, maybe?

File: assets/lang/ru_ru.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "index": "Главная",
+    "contents": "Содержание",
+    "version": "Версия",
+    "api_docs": "Документация API",
+    "github": "GitHub",
+    "discord": "Discord",
+    "categories": "Категории",
+    "home": "Это перевод Полевого справочника TerraFirmaCraft, который можно просмотреть в игре. Некоторые части этого справочника видны только в игре, такие как визуализации мультиблоков или некоторые рецепты.",
+    "multiblock": "Мультиблок",
+    "multiblock_only_in_game": "Просмотрите полевой справочник в Minecraft, чтобы увидеть этот мультиблок.",
```
Comment (DarthSidiousPalpatine) [inline assets/lang/ru_ru.json]: Просмотреть - means read it whole. You need Посмотреть.

File: assets/lang/ru_ru.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "title": "Полевой справочник TerraFirmaCraft",
```
Comment (DarthSidiousPalpatine) [inline assets/lang/ru_ru.json]: Are you... using google translate? I mean, you can translate Field Guide not directly as Полевой справочник, which sounds weird, but like Путеводитель? Or maybe just справочник.

File: assets/lang/ru_ru.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "index": "Главная",
+    "contents": "Содержание",
+    "version": "Версия",
+    "api_docs": "Документация API",
+    "github": "GitHub",
+    "discord": "Discord",
+    "categories": "Категории",
+    "home": "Это перевод Полевого справочника TerraFirmaCraft, который можно просмотреть в игре. Некоторые части этого справочника видны только в игре, такие как визуализации мультиблоков или некоторые рецепты.",
+    "multiblock": "Мультиблок",
```
Comment (DarthSidiousPalpatine) [inline assets/lang/ru_ru.json]: Again, Мультиблок in sentence sounds weird, Try use Структура.

File: assets/lang/ru_ru.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+    "title": "Полевой справочник TerraFirmaCraft",
+    "index": "Главная",
+    "contents": "Содержание",
+    "version": "Версия",
+    "api_docs": "Документация API",
+    "github": "GitHub",
+    "discord": "Discord",
+    "categories": "Категории",
+    "home": "Это перевод Полевого справочника TerraFirmaCraft, который можно просмотреть в игре. Некоторые части этого справочника видны только в игре, такие как визуализации мультиблоков или некоторые рецепты.",
```
Comment (DarthSidiousPalpatine) [inline assets/lang/ru_ru.json]: Unnecessary capital letter in Полевого.
And Просмотреть means read it whole at once. You need посмотреть.

Тавтология, Некоторые и некоторые.

File: assets/lang/ru_ru.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    "multiblock": "Мультиблок",
+    "multiblock_only_in_game": "Просмотрите полевой справочник в Minecraft, чтобы увидеть этот мультиблок.",
+    "recipe": "Рецепт",
+    "recipe_only_in_game": "Просмотрите полевой справочник в Minecraft, чтобы увидеть этот рецепт.",
+    "item": "Предмет",
+    "items": "Предметы",
+    "item_only_in_game": "Просмотрите полевой справочник в Minecraft, чтобы увидеть этот предмет.",
+    "addon": "(Дополнение) %s",
+    "ticks": "%s Тика",
```
Comment (DarthSidiousPalpatine) [inline assets/lang/ru_ru.json]: This is funny cuz you cant directly translate %s ticks to russian in one case.
Your Тика will be correct only for numbers %2, %3, %4.

Maybe let it be as ticks or translate as тиков or make "тиков: (%s)"?

**Review by DarthSidiousPalpatine — COMMENTED** [review]
I'm not a dev, but i have some experience in translation for games and I am russian, so... here is my little review. ¯\_(ツ)_/¯

**Comment by NaoCraftLab** [comment]
@DarthSidiousPalpatine, thank you for the review! I've made the corrections. Could you please check them?

**Pull request merged**